coralproject/talk
# [next] Tabs Components - Tab, TabBar, TabPane, TabContent

## What does this PR do?
- Adds `Tab`, `TabBar`, `TabContent`, `TabPane`

#### Proposed API
```jsx
<TabBar color="secondary" activeTab={props.activeTab} onTabClick={(tab)=> props.setActiveTab(tab)}>
  <Tab tabId="one">One</Tab>
  <Tab tabId="two">Two</Tab>
  <Tab tabId="three">Three</Tab>
</TabBar>
<TabContent activeTab={props.activeTab}>
  <TabPane tabId="one">Hola One</TabPane>
  <TabPane tabId="two">Hola Two</TabPane>
  <TabPane tabId="three">Hola Three</TabPane>
</TabContent>
```

## How do I test this PR?
- Run docz 
- Check [http://localhost:3030/Tabs](http://localhost:3030/Tabs)
